### DISCUSSION TITLE
Cant connect to SynScan Azmount

### DISCUSSION THREAD

**Discussion by patrober** [Stellarium/stellarium#3026] [General]
Newbie here, and apologies if this is the wrong place/forum. 

Been bashing my head around trying to connect Stellarium to my SynScan Az mount. I have successfully connected it to the Skywatcher pc app and can control it via that, so the cable, drivers, com port are correct. But when I repeat the same with Stellarium it does nothing. 

Ive setup the correct mount/controller (Sky-Watcher Synscan AZ Goto)  and com port (port 5), have the synscan in PC direct mode, but get the following error message in the log-

ERROR creating TelescopeClientDirectNexStar: cannot open serial device "COM5"
TelescopeClient::create(): Unable to create a telescope client.

Any ideas, as Im struggling what to try next.

**Comment by gzotti** [comment]
See the many open issues around TelescopeControl plugin.  We have an open issue running concerning COM port. 
See also the preliminary solution/escape given in the FAQ in the wiki.

**Reply by patrober** [reply]
Thanks - I did search for the problem, but my search was too specific alas (only searched for problems with the same mount) so never saw that. 

Thanks again.

**Reply by patrober** [reply]
> See the many open issues around TelescopeControl plugin. We have an open issue running concerning COM port. See also the preliminary solution/escape given in the FAQ in the wiki.

Had a look at the faq and couldn't see anything that seemed to cover this. There is a section elsewhere in the wiki about ASCOM  control. Did you mean that?

**Reply by gzotti** [reply]
Was not completely clear. https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#problems-with-telescopecontrol-plugin

**Reply by patrober** [reply]
> Was not completely clear. https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#problems-with-telescopecontrol-plugin

thanks for that. Installed the ASCOM drivers and software and that seems to work with Stellarium. Sorted!  Thanks.
